packages/storybook-ui-components/stories/1-Badge.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+import { Badge } from "@cockroachlabs/ui-components";
+
+import m from "../utils/selectKnobMirror";
+
+export default {
+  title: "Badge",
+  components: Badge,
+  decorators: [withKnobs],
+};
+
+export const Demo = () => (
+  <Badge
+    content={text("Content Prop", "badge")}
+    intent={select(
+      "Intent",
+      m(["neutral", "success", "warning", "danger"]),
```

Comment:
The `m` is temporary. That was just me being lazy. This should be a name that increases the readability here.